mitmproxy/optmanager.py

```diff
@@ -83,11 +82,19 @@ class _UnconvertedStrings:
     val: list[str]
 
 
+def _sig_changed(updated: set[str]) -> None:
```

Comment:
Maybe it might be a bit cleaner to define signature functions as inner functions in `__init__` before using them in `SyncSignal` constructor?

Reply:
Unfortunately that looks ugly due to the extensive whitespace around it (which is enforced by black). I've made it a bit clearer what they are for. :)